app/code/Transoft/Blog/view/adminhtml/layout/blog_post_newaction.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0"?>
```

Comment:
It seems that we don't need this. new action redirects to edit

Reply:
removed